src/activities/nine_men_morris/NineMenMorris.qml

```diff
@@ -387,170 +325,35 @@ ActivityBase {
                 left: background.left
                 leftMargin: 5
             }
-            radius: 5
             state: "second"
             visible: !items.isTutorial
-
-            Image {
-                id: player1background
-                source: Activity.url + "score_1.svg"
-                sourceSize.height: parent.height * 0.93
-                anchors.centerIn: parent
-
-                Image {
-                    id: player1image
-                    source: Activity.url + "KonqiWhite.svg"
-                    sourceSize.height: parent.height * 0.8
-                    x: parent.width * 0.05
-                    anchors.verticalCenter: parent.verticalCenter
-                }
-
-                GCText {
-                    id: player1_score
-                    anchors.verticalCenter: parent.verticalCenter
-                    color: "#2a2a2a"
-                    x: parent.width * 0.65
-                    fontSize: mediumSize
-                }
-            }
-
-            states: [
-                State {
-                    name: "first"
-                    PropertyChanges {
-                        target: player1image
-                        source: Activity.url + "KonqiWhite.svg"
-                    }
-                    PropertyChanges {
-                        target: player1
-                        color: "#f07c49"
-                    }
-                },
-                State {
-                    name: "second"
-                    PropertyChanges {
-                        target: player1
-                        color: "transparent"
-                    }
-                    PropertyChanges {
-                            target: player1image
-                            source: Activity.url + "KonqiWhite.svg"
-                    }
-                },
-                State {
-                    name: "win"
-                    PropertyChanges {
-                        target: player1image
-                        source: Activity.url + "win.svg"
-                    }
-                    PropertyChanges {
-                        target: player1
-                        color: "#f7ec5d"
-                    }
-                }
-            ]
-
-            transform: Scale {
-                id: changeScalePlayer1
-                property real scale: 1
-                xScale: scale
-                yScale: scale
-            }
+            playerImageSource: Activity.url + "KonqiWhite.svg"
+            backgroundImageSource: Activity.url + "score_1.svg"
         }
         // Player scores section ends
 
         // Animation section start
         PropertyAnimation {
             id: player1turn
```

Comment:
I'm not sure this animation should exist anymore, if I'm not wrong, it's handled in the ScoreItem